src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -254,6 +254,11 @@ internal static int Start(string bannerText, string helpText)
                 {
 #if LEGACYTELEMETRY
                     TelemetryAPI.ReportExitTelemetry(s_theConsoleHost);
+#endif
+#if UNIX
+                    // https://github.com/dotnet/runtime/issues/27626 leaves terminal in application mode
+                    // for now, we explicitly emit DECRST 1 sequence
+                    s_theConsoleHost.UI.Write("\x1b[?1l");
```

Comment:
Should we check VT mode?

Reply:
Yes, also it seems that we should only emit for interactive

Reply:
if `pwsh -noninteractive` is used, we should not emit as it gets added to the output which is what causes the build to fail